cluster-operator/Dockerfile

```diff
@@ -5,6 +5,5 @@ ENV STRIMZI_VERSION ${strimzi_version}
 
 ADD target/cluster-operator-${strimzi_version}.jar /
 
-USER 1001
-
+USER strimzi:root
```

Comment:
I would need to search the history. But using named users was causing problems and therefore we swicthed to just IDs. So I think you should revert this back here as well as in all the other places.